jwql/website/apps/jwql/data_containers.py

```diff
@@ -790,6 +790,58 @@ def get_proposal_info(filepaths):
     return proposal_info
 
 
+def get_thumbnails_all_instruments(instruments):
+    """Return a list of thumbnails available in the filesystem for all
+    instruments given requested parameters.
+
+    Returns
+    -------
+    thumbnails : list
+        A list of thumbnails available in the filesystem for the
+        given instrument.
+    """
+
+    # Make sure instruments are of the proper format (e.g. "Nircam")
+    thumbnail_list = []
+    for inst in instruments:  # JWST_INSTRUMENT_NAMES:
+        instrument = inst[0].upper()+inst[1:].lower()
```

Comment:
It is unfortunately that MAST requires this naming convention, it can get messy!